mypy/test/data.py

```diff
@@ -65,6 +65,8 @@ def parse_test_cases(
             tcout = []  # type: List[str]
             if i < len(p) and p[i].id == 'out':
                 tcout = p[i].data
+                if p[i].arg == 'pathfix':
+                    tcout = [s.replace('/', os.path.sep) for s in tcout]
```

Comment:
Rather than having to remember to add 'pathfix' to tests that happen to have pathnames with slashes in the expected output, maybe you could always fix the path in the text up to the first colon, skipping this completely if the line starts with whitespace or there's whitespace in the stuff before that first colon (or there is no colon)?